reana_commons/config.py

```diff
@@ -117,6 +117,21 @@ def load(cls, kind):
 By default runtime pods will run in the same namespace as the infrastructure pods.
 """
 
+REANA_RUNTIME_KUBERNETES_NODE_LABEL = (
+    {
+        os.getenv("REANA_RUNTIME_KUBERNETES_NODE_LABEL")
+        .split("=")[0]: os.getenv("REANA_RUNTIME_KUBERNETES_NODE_LABEL")
+        .split("=")[1]
+    }
+    if os.getenv("REANA_RUNTIME_KUBERNETES_NODE_LABEL")
+    else {}
+)
+"""Kubernetes label (with format ``lable_name=lable_value``) which identifies the nodes where the runtime pods should run.
```

Comment:
```suggestion
"""Kubernetes label (with format ``label_name=label_value``) which identifies the nodes where the runtime pods should run.
```

Reply:
Perhaps add a release commit afterwards.  Is there anything coming to `r-commons` shortly?

Reply:
I don't think we need anything else than this (which was "unexpected"). I will add the commit.